dss/api/health.py

```diff
@@ -0,0 +1,78 @@
+import datetime
+import os
+import boto3
+from botocore import errorfactory
+import requests
+from flask import json
+import sys
+
+from dss.index.es import ElasticsearchClient
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), 'chalicelib'))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from googleapiclient import discovery
+from dss.util.aws.clients import dynamodb  # type: ignore
+
+
+def _get_es_status(host: str = "localhost", port: int = None):
+    es_status = False
+    if port is not None:
+        es_client = ElasticsearchClient().get()
+    else:
+        es_client = ElasticsearchClient()._get(host, port, 1)
+    es_res = es_client.cluster.health()
+    if es_res['status'] == 'green':
+        es_status = True
+    return es_status, es_res
+
+
+def _get_dynamodb_status():
+    db_status = True
+    ddb_tables = list(filter(lambda x: x.startswith("dss") and x.endswith(os.getenv("DSS_DEPLOYMENT_STAGE")),
+                             dynamodb.list_tables()["TableNames"]))
```

Comment:
The criteria used to find dynamo-db tables is not exclusive to the data-store. For now, perhaps we should use an explicit list.

Reply:
changed to explicit list